airbyte-webapp/src/packages/cloud/lib/domain/cloudWorkspaces/types.ts

```diff
@@ -17,6 +19,8 @@ export interface CloudWorkspace {
 
 export interface CreditConsumptionByConnector {
   connectionId: string;
+  connectionName: string;
+  status: ConnectionStatus;
```

Comment:
New data added to this endpoint in https://github.com/airbytehq/airbyte-platform-internal/pull/4164